tools/wptrunner/wptrunner/testrunner.py

```diff
@@ -563,10 +563,12 @@ def test_ended(self, test, results):
         # TODO: consider changing result if there is a crash dump file
 
         # Write the result of the test harness
+        result_subns = {"INTERNAL-ERROR": "ERROR",
+                        "EXTERNAL-TIMEOUT": "TIMEOUT"}
```

Comment:
The word "result" is being applied to the object that contains this status string. What do you think of `status_subns` or `status_map`?